pkg/compact/downsample/downsample.go

```diff
@@ -289,7 +289,13 @@ func (b *aggrChunkBuilder) add(t int64, aggr *aggregator) {
 	b.added++
 }
 
-func (b *aggrChunkBuilder) finalizeChunk(lastT int64, trueSample float64) {
+func (b *aggrChunkBuilder) firstRawSample(firstT int64, trueSample float64) {
```

Comment:
those functions are quite shallow, and really the same. Can we maybe just inline with the comment? We use it twice, sure, but if we would inline them it might be even clearer?

Reply:
Maybe but IMHO this split up is clear too since literally the function's name tells you what's happening :smile: up to you.